docs/yamlpages/protocols.rst

```diff
@@ -0,0 +1,66 @@
+.. _yaml_protocols_head:
+
+Protocols Header for YAML Files
+*******************************
+
+The ``protocols`` header tells YANK which alchemical thermodynamic path to draw samples along. Together with a :doc:`system <systems>`, this header makes a complete :doc:`experiment <experiments>`.
+
+Just like in :doc:`molecules <molecules>`, protocols can have arbitrary, user defined names.
+In the examples, these user defined names are marked as ``{UserDefinedProtocol}``.
+
+
+----
+
+
+.. _yaml_protocols_example:
+
+Protocols Syntax
+================
+.. code-block:: yaml
+
+   protocols:
+     {UserDefinedProtocol:
+       {PhaseName1}:
+         alchemical_path:
+           lambda_electrostatics: [1.00, 0.75, 0.50, 0.25, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
+           lambda_sterics:        [1.00, 1.00, 1.00, 1.00, 1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.50, 0.40, 0.30, 0.20, 0.10, 0.00]
+       {PhaseName2}:
+         alchemical_path:
+           lambda_electrostatics: [1.00, 0.75, 0.50, 0.25, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
+           lambda_sterics:        [1.00, 1.00, 1.00, 1.00, 1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.50, 0.40, 0.30, 0.20, 0.10, 0.00]
+
+This is the only valid format for the protocols header and consistes of multiple parts which we break down here.
+
+The ``{PhaseName1}`` and ``{PhaseName2}`` headers are ``system`` dependent headers. 
+The ``{UserDefinedSystem`` and ``{UserDefinedProtocol}`` are combined in the ``experiements`` header to make on object.
+Each ``{PhaseNameX}`` in the ``{UserDefinedProtocol}`` must match with the expected type of system created. 
+Below is the mappings of each ``{PhaseNameX}``:
+
+* :ref:`Ligand/Receptor Free Energies Setup by YANK <yaml_systems_receptor_ligand>`
+
+  * ``{PhaseName1}`` -> ``complex``
+  * ``{PhaseName2}`` -> ``solvent``
+
+* :ref:`Hydration Free Energies Setup by YANK <yaml_systems_hydration>`
+
+  * ``{PhaseName1}`` -> ``solvent1``
+  * ``{PhaseName2}`` -> ``solvent2``
+
+* :ref:`Arbitrary Phase Free Energies Setup by User <yaml_systems_user_defined>`
+
+  * ``{PhaseName1}`` -> ``phase1``
+  * ``{PhaseName2}`` -> ``phase2``
```

Comment:
Ok, I did not document very well phase names, sorry. The phase names don't have to match exactly those names as long if they contains those keywords. For example, 'my_complex_phase' and 'blablasolventbla` are accepted and they match a binding free energy calculation. It is also possible to specify any name if you use the `!Ordered` keyword (I've enforced this to make it clear to the user that the order of the phases determines the sign of the result). For example
```yaml
absolute-binding: !Ordered
  plus_sign_phase:
    alchemical_path: ...
  minus_sign_phase:
    alchemical_path: ...
```